example/src/Main.purs

```diff
@@ -280,31 +285,31 @@ example =
         H.modify_ \s → s{ intervals = insert idx (show val) s.intervals }
     pure next
 
-
-
 type ExampleConfig fmtInput input fmt query out m =
   { mkFormat ∷ fmtInput → StrOr fmt
   , unformat ∷ fmt → String → StrOr input
   , picker ∷ fmt → H.Component HH.HTML query Unit out m
   , handler ∷ ∀ z. Int → out → z → Query z
   , setter ∷ ∀ z. Int → Maybe input → z → Query z
-  , cp ∷ CP.ChildPath query ChildQuery Int Slot
   }
```

Comment:
First tried to get it working with `cp` in, but couldn't work out `renderExample` with it. Happy to try to get it back in if there are any ideas, although it didn't work out that bad leaving it out either.